Formula/cassandra@3.rb

```diff
@@ -0,0 +1,154 @@
+class CassandraAT3 < Formula
+  desc "Eventually consistent, distributed key-value store"
+  homepage "https://cassandra.apache.org"
+  url "https://www.apache.org/dyn/closer.lua?path=cassandra/3.11.10/apache-cassandra-3.11.10-bin.tar.gz"
+  mirror "https://archive.apache.org/dist/cassandra/3.11.10/apache-cassandra-3.11.10-bin.tar.gz"
+  sha256 "bbe772956c841158e3228c3b6c8fc38cece6bceeface695473c59c0573039bf1"
+  license "Apache-2.0"
+
+  keg_only :versioned_formula
+
+  deprecate! date: "2022-04-01", because: :deprecated_upstream
+
+  depends_on "cython" => :build
```

Comment:
I'm a bit surprised this is used, since `cython` uses Python 3.9.

Reply:
I took the formula as it was before Cassandra 4.0 was made available in Homebrew.

Reply:
I'm aware. I would ask the same question about the last version of the formula.